src/PrestaShopBundle/Form/Admin/Configure/ShopParameters/TrafficSeo/Meta/SetUpUrlType.php

```diff
@@ -52,44 +53,92 @@ class SetUpUrlType extends AbstractType
      */
     private $isHostMode;
 
+    /*
+    * @var Tools
+    */
+    private $tools;
+
     /**
      * SetUpUrlType constructor.
      *
+     * @param \Symfony\Component\Translation\TranslatorInterface $translator
+     * @param array $locales
      * @param array $canonicalUrlChoices
      * @param bool $isHtaccessFileWritable
      * @param bool $isHostMode
+     * @param $tools
      */
     public function __construct(
+        TranslatorInterface $translator,
+        array $locales,
         array $canonicalUrlChoices,
         $isHtaccessFileWritable,
-        $isHostMode
+        $isHostMode,
+        $tools
     ) {
+        parent::__construct($translator, $locales);
         $this->canonicalUrlChoices = $canonicalUrlChoices;
         $this->isHtaccessFileWritable = $isHtaccessFileWritable;
         $this->isHostMode = $isHostMode;
+        $this->tools = $tools;
     }
 
     /**
      * {@inheritdoc}
      */
     public function buildForm(FormBuilderInterface $builder, array $options)
     {
+        $friendlyUrlHelp = $this->trans(
+            'Enable this option only if your server allows URL rewriting (recommended).',
+            'Admin.Shopparameters.Help'
+        );
+
+        if (!$this->tools->isModRewriteActive()) {
+            $friendlyUrlHelp .=
+                '</br>' . $this->trans(
+                'URL rewriting (mod_rewrite) is not active on your server, or it is not possible to check your server configuration. If you want to use Friendly URLs, you must activate this mod.',
```

Comment:
</br> works as intended but if there is a better or cleaner way to add new line I would love to hear a suggestion.

Reply:
you mean "mod" in 

> you must activate this mod.

Right? In that case, yeah it seems so. String was exactly the same before and the apache module is called "mod_rewrite"

Reply:
Ok thank you I just wanted to be sure :)